modules/RTC/TraceablePeerConnection.js

```diff
@@ -1852,6 +1875,21 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(
         });
     }
 
+    if (this.options.enableLayerSuspension) {
```

Comment:
do you mind describing this new option in TraceablePeerConnection constructor like few others are ?

Reply:
nice catch, fixed